ci-scripts/rhel8SanityCheckReport.py

```diff
@@ -342,28 +366,38 @@ def testSummaryDetails(self):
 			details += self.addDetailsRow('SPGWC SX Assoc Response', res['spgwc_answer'], 'IP = ' + res['spgwc_ipv4'])
 			if not res['spgwu_request'] or not res['spgwc_answer']:
 				status = False
+				print ('SX Association => KO')
+			else:
+				print ('SX Association => OK')
 			# SPGWC to SPGWU HeartBeat
 			res = check_cups_heartbeat('archives/' + pcap_fil)
 			details += self.addDetailsRow('SPGWC SX HRT Request', res['spgwc_hrt_request'], 'NB REQ = ' + res['spgwc_hrt_nb_req'])
 			details += self.addDetailsRow('SPGWU SX HRT Response', res['spgwu_hrt_answer'], 'NB RES = ' + res['spgwu_hrt_nb_res'])
 			if not res['spgwc_hrt_request'] or not res['spgwu_hrt_answer']:
 				status = False
+				print ('SX Heartbeat => KO')
+			else:
+				print ('SX Heartbeat => OK')
 		if os.path.isfile('archives/ping_trf_gen_from_ue.log') or os.path.isfile('archives/ping_ue_from_trf_gen.log'):
 			details += self.addSectionRow('Ping Traffic Test')
 			res = self.checkPingLog('archives/ping_trf_gen_from_ue.log')
 			if res['status']:
-				print('Ping From UE : OK')
+				print('Ping From UE => OK')
```

Comment:
@rdefosse Changing from : to => won't affect the parsing in HTML at any place? Just asking to be sure